Added licence

kesha-antonov opened the pull request.

kesha-antonov committed:
Create LICENCE

kesha-antonov committed:
Update package.json

aksonov commented:
Should definitely be Facebook copyright because it is just fork of NavigationExperimental API. Also there is typo - LICENSE

kesha-antonov committed:
Update and rename LICENCE to LICENSE

added fb license from https://github.com/facebook/react-native/blob/master/LICENSE
